### PR TITLE
C#: recognize Operators as such, fix split-up numeric literals

### PR DESCRIPTION
**Pull request by birkenfeld**

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -23,6 +23,7 @@ Version 2.14.0
   * C family (C, C++ and many others): fix an issue where a chunk would
     be wrongly recognized as a function definition due to braces in
     comments (#2210)
+  * C#: fix number and operator recognition (#2256)
```
[the remark this answers is not in the document]
Comment (birkenfeld) [reply CHANGES:26]: So better PR number instead of issue number?

[the remark this answers is not in the document]
Comment (Anteru) [reply CHANGES]: I add both the issues and the PR to all entries in `CHANGES`.

[the remark this answers is not in the document]
Comment (Dreamypops33) [reply CHANGES]: 2256 dreamypops

**Comment by Dreamypops33** [comment]
Sorry for the delay

**Pull request merged**